experiments/depth-series/depth_session.py

```
@@ -0,0 +1,134 @@
+        if not base.wait_socket(cfg): raise SystemExit("the gate socket never stood")
+        for i, group in enumerate(turns, 1):
+            body = "\n\n".join(open(p).read() for p in group)
+            t0 = time.time(); answer = base.gate_turn(cfg, f"{ASK}\n\n{body}", timeout=900); dt = time.time() - t0
+            entry = {"turn": i, "pages": [os.path.basename(p) for p in group], "input_words": len(body.split()),
+                     "seconds": round(dt, 1), "answer_kind": answer.get("kind"), "turn_key": answer.get("turn"), "run": answer.get("run")}
+            report["turns"].append(entry); print(json.dumps(entry), flush=True)
+            # **An unanswered turn ends the run**, recorded as such: a report
+            # with fewer valid turns that read as complete would be the
+            # defect the deposit exists to prevent.
+            if answer.get("kind") != "answered":
+                failed = {"turn": i, "answer": answer}; report["failed"] = failed
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)